### PR TITLE
master

### PR DESCRIPTION
**Pull request by DovAmir**
add node and container details, this is how the result looks
https://app.box.com/s/9tnx9dq7ipmbdfaxlqcb7xnnq0i7cup6

### CODE REVIEW

File: .idea/uiDesigner.xml [hunk 1]
```diff
@@ -0,0 +1,124 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (ManoMarks) [inline .idea/uiDesigner.xml]: Is this a necessary file? I don't see it referenced anywhere and looks like it's a Java file

File: build_run.sh [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#RUN npm install --unsafe-perm=true
```
Comment (ManoMarks) [inline build_run.sh]: I noticed that you added a .gitignore that specified .sh files, did you mean to add these?

**Comment by ManoMarks** [comment]
Added a couple of comments on particular files that look like they're extraneous.

File: .idea/uiDesigner.xml (hunk 1 above)
Comment (DovAmir) [inline .idea/uiDesigner.xml]: .idea/uiDesigner.xml is not needed

File: build_run.sh (hunk 2 above)
Comment (DovAmir) [inline build_run.sh]: **build.sh** creates a local swarm and runs the visualizer localy for testing.

**push.sh** is a script that pushes to dockerhub and should be **removed** from the repo (it points to my own dockerhub repo)

**Pull request closed without merging**